### DISCUSSION TITLE
Echelon Watt Table

### DISCUSSION THREAD

**Discussion by wbjenn** [cagnulein/qdomyos-zwift#483] [Q&A]
Which Echelon bike was used to create the master watt table used for QZ? Thanks.

**Comment by cagnulein** [comment]
Ex3 if I remember correctly

Il giorno mer 3 nov 2021 alle 22:19 wbjenn ***@***.***> ha
scritto:

> Which Echelon bike was used to create the master watt table used for QZ?
> Thanks.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/discussions/483>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWAJ5WI5CCN2OMN4LH3UKGYPHANCNFSM5HJ6YRHA>
> .
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud

**Reply by wbjenn** [reply]
The table suggests the Echelon Sport Connect. It would be good to know for sure which was used. How can we get a message to @mgarcea?

**Reply by cagnulein** [reply]
He will see the notification, don't worry:)

**Reply by mgarcea** [reply]
Our watt table (not the master) was created using Vector Power Pedals and the Costco EX-4S which I believe uses the same fly wheel and resistance as the EX-5 and the EX-3.  The Sport Connect is different and has much easier resistance at the top end. I'm not sure what the master was created on, but in testing, I did notice that the few values on QZ at a given cadence and resistance that I checked were the same as those on the Echelon app in Freeride mode with our machine.  I didn't test a lot of values, and it was quite a while ago, so I would check and see if the QZ master table matches up to your bike and the Echelon app in Freeride Mode at the higher resistance levels.

**Reply by wbjenn** [reply]
@mgarcea Thanks for the response. I have an EX-3 and am noticing some oddities at the higher resistance levels (esp. 30-32). I've never used my current bike with the Echelon app, so I'll give it a try and see how the wattage responds. For creating the table, is it increments of 10rpm at each resistance level? Thanks.

**Reply by asoltani** [reply]
@mgarcea how did you generate the wattage tables?  i'm planning to test an EX3 and EX4S using a set of Favero powermeter pedals . I'd like to generate outputs for both bikes for reference as they 'feel' drastically different at a given resistance setting.

**Reply by mgarcea** [reply]
Interesting, curious to know the results of your tests.  We used an EX4S. Some ideas:

Captured Power of the pedals using Wahoo App.
Captured Echelon info (Power, cadence) using the Qdomyos App 
Used cadences 50, 60, 70, 80, 90, 100
Road as consistently as possible at those cadences for 30s to 1 min at a specific resistance setting on the bike or as long as needed to get a relatively consistent reading.
We didn't test lower than resistance 10 on the Echelon
Imported both files into Golden Cheetah for analysis and applied some smoothing (2-5s avg) to the charts
Took those new observed Power values and used regression to obtain the values at other cadences in between.
It wasn't perfect, but closer.
There is a definite issue with resistance falling off the longer you ride the bike.
You could also video and look at the comparison, but charting it allows you to smooth the data and see the differences between the two charts.

**Reply by mgarcea** [reply]
You end up with something that looks like this in Golden Cheetah.  

![image](https://user-images.githubusercontent.com/84399684/217028587-3f450fc7-f680-445a-ae93-fb070a971c61.png)

**Comment by cagnulein** [comment]
Yeah sorry, i didn't read well the question @wbjenn The master table was for a Echelon Sport Connect.
